aspnetcore/mvc/views/tag-helpers/built-in/AnchorTagHelper.md

```diff
@@ -182,7 +182,7 @@ The domain in the example is localhost, but the Anchor Tag Helper uses the websi
 
 - - -
 
-## Additional Resources
+## Resources
```

Comment:
They had a meeting and decided not to use "Resources" after all, but the "r" should be lowercase.

```
## Additional resources
```